### PR TITLE
#2045 New angular p.u. types

### PR DESCRIPTION
**Pull request by ErwanGuichard49**

### CODE REVIEW

File: dynawo/sources/Models/Modelica/Dynawo/Types.mo [hunk 1]
```diff
@@ -65,5 +65,9 @@ package Types "Standard types for electrical variables"
   // Percent
   type Percent = Real (unit = "100") "Percent quantity";
 
+  type AngularVelocityPu = PerUnit;
```
Comment (cossartque) [inline dynawo/sources/Models/Modelica/Dynawo/Types.mo:68]: Could you put it with the other per unit types ?

**Review by cossartque — CHANGES_REQUESTED** [review]

File: dynawo/sources/Models/Modelica/Dynawo/Types.mo (hunk 1 above)
Comment (ErwanGuichard49) [reply dynawo/sources/Models/Modelica/Dynawo/Types.mo:68]: Yes. Moreover, should I put the pu types in alphabetical order ?

Comment (cossartque) [reply dynawo/sources/Models/Modelica/Dynawo/Types.mo:68]: Not necesary

File: dynawo/sources/Models/Modelica/Dynawo/Types.mo [hunk 2]
```diff
@@ -63,7 +63,11 @@ package Types "Standard types for electrical variables"
   type ApparentPowerPu = PerUnit;
 
   // Percent
-  type Percent = Real (unit = "100") "Percent quantity";
+  type AngularVelocityPu = PerUnit;
```
Comment (cossartque) [inline dynawo/sources/Models/Modelica/Dynawo/Types.mo:66]: Could you move them just after ApparentPowerPu ? Maybe I was not clear

Comment (cossartque) [reply dynawo/sources/Models/Modelica/Dynawo/Types.mo:66]: It is not a Percent

Comment (ErwanGuichard49) [reply dynawo/sources/Models/Modelica/Dynawo/Types.mo:66]: Already done.

**Review by cossartque — APPROVED** [review]

**Pull request merged**